core-java-modules/core-java-15/src/main/java/whatsnew/records/Person.java

```diff
@@ -0,0 +1,16 @@
+package whatsnew.records;
```

Comment:
Let's use com.baeldung.whatsnew for the package

Reply:
Just pushed an updated branch with updated package names